packages/web/src/components/nav/desktop/PlaylistFolderNavItem.tsx

```diff
@@ -68,25 +69,36 @@ const FolderNavLink = ({
   )
 }
 
-export const PlaylistFolderNavItem = ({
-  folder,
-  hasUpdate = false,
-  dragging,
-  draggingKind,
-  onClickEdit
-}: {
+type PlaylistFolderNavItemProps = {
   folder: PlaylistLibraryFolder
   hasUpdate: boolean
   dragging: boolean
   draggingKind: string
   onClickEdit: (folderId: string) => void
-}) => {
-  const { id, name, contents } = folder
+  children?: React.ReactNode
+}
+
+export const PlaylistFolderNavItem = ({
+  folder,
+  hasUpdate = false,
+  dragging,
+  draggingKind,
+  onClickEdit,
+  children
+}: PlaylistFolderNavItemProps) => {
+  const { id, name } = folder
   const isDroppableKind =
     draggingKind === 'track' ||
     draggingKind === 'playlist' ||
     draggingKind === 'playlist-folder'
   const [isHovering, setIsHovering] = useState(false)
+  const [isExpanded, setIsExpanded] = useState(false)
+  const [ref, bounds] = useMeasure()
+  const contentsStyle = useSpring({
+    height: isExpanded ? bounds.height : 0,
+    opacity: isExpanded ? 100 : 0,
+    overflow: 'hidden'
+  })
```

Comment:
Planning on making a `<Collapse>` utility component for Stems using this pattern and then will just use that component here https://linear.app/audius/issue/AUD-1664/add-collapse-utility-component